doc/source/user_guide/ug_implementation.rst

```diff
@@ -571,7 +578,62 @@ to set the initial model time to the namelist values defined by
 Normally, ``use_restart_time`` is set to false on the initial run
 and then set to true on subsequent restart runs of the same
 case to allow time to advance thereafter.  More information about 
-the restart capability can be found here, :ref:`restartfiles`.
+the restart capability can be found in :ref:`restartfiles`.
+
+Several different calendars are supported including noleap (365 days
+per year), 360-day (twelve 30 day months per year), and gregorian
+(leap days every 4 years except every 100 years except every 400
+years).  The gregorian calendar in CICE is formally a proleptic gregorian
+calendar without any discontinuties over time.  The calendar is set
+by specifying ``days_per_year`` and ``use_leap_years`` in the
+namelist, and the following combinations are supported,
+
+.. _tab-cal:
+
+.. table:: Supported Calendar Options
+
+   +----------------+----------------------+-----------------------+
+   | calendar       | days_per_year        |  use_leap_years       |
+   +================+======================+=======================+
+   | noleap         | 365                  |  false                |
+   +----------------+----------------------+-----------------------+
+   | gregorian      | 365                  |  true                 |
+   +----------------+----------------------+-----------------------+
+   | 360-day        | 360                  |  false                |
+   +----------------+----------------------+-----------------------+
+
+
+The history (:ref:`history`) and restart (:ref:`restartfiles`) 
+outputs and frequencies are specified in namelist and
+are computed relative to a reference date defined by the namelist
+``histfreq_base`` and ``dumpfreq_base``.  Valid values for each are 
+``zero`` and ``init``.  If set to ``zero``, all output will be relative 
+to the absolute reference date, 0000-01-01-00000.  This is the default
+value for ``histfreq_base``, so runs with different initial
+dates will have identical output.  If set to ``init``, all frequencies
+will be relative to the model init time specified by ``year_init``,
+``month_init``, and ``day_init``.  This is the default for
```

Comment:
Maybe we would want to emphasize specifically that `sec_init` is _not_ taken into account ?

Reply:
Done.